standalone-metastore/pom.xml

```diff
@@ -58,6 +58,7 @@
     <maven.surefire.plugin.version>3.0.0-M4</maven.surefire.plugin.version>
     <!-- Dependency versions -->
     <antlr.version>3.5.2</antlr.version>
+    <antlr4.version>4.8-1</antlr4.version>
```

Comment:
Wouldn't be possible to use the 4.13? As we upgraded, would be much better to go the latest if it is possible.

Reply:
The ANTLR version beyond 4.9.3 requires JDK 11, since Hive still use JDK 8, so we can update to 4.9.3

Reply:
I think it is time to think after 4.0 release to move ahead with JDK version not just on runtime side.

Reply:
Agreed!